service/recognition/noise_reduction_model.py

```diff
@@ -0,0 +1,35 @@
+"""
+Noise-reduction module
+"""
+
+from typing import Any
+
+import torch
+from df.enhance import enhance, init_df
+
+from .model import RecognitionModel
+
+
+class NoiseReductionModel(RecognitionModel):
+    """
+    Class for noise-reduction model
+    """
+    def __init__(self):
+        self.model = self.load_model()[0]
+        self.df_state = self.load_model()[1]
+
+    def load_model(self) -> Any:
```

Comment:
Can you add the ability to customize the model through the constructor? You can pay attention to how it was done in the vosk model

```python
class VoskModel(Recognition Model):
    def __init__(self, model: Model = Model(lang="ru"), buff_size: int = 4000):
    ...
```

Is there an opportunity to implement this?